backend/core/CalibrationEngine.py

```
@@ -0,0 +1,657 @@
+            
+            return vol
+        
+        def objective(params, F_values, K_values, T_values, observed_vols):
+            alpha, beta, rho, nu = params
+            try:
+                fitted_vols = np.array([
+                    sabr_vol(F, K, T, alpha, beta, rho, nu) 
+                    for F, K, T in zip(F_values, K_values, T_values)
+                ])
+                return np.sum((fitted_vols - observed_vols)**2)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)